projects/aws/eks-a-admin-image/provisioners/install_kind.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# KIND_VERSION needs to be provided to the script
+
+KIND_URL=https://kind.sigs.k8s.io/dl/${KIND_VERSION}/kind-linux-amd64
```

Comment:
id rather that we install our build of kind, but its not a huge deal.

Reply:
I didn't know we had our own. Do you have link where I can get it?